src/earn/EarnHome.tsx

```diff
@@ -238,7 +238,7 @@ export default function EarnHome({ navigation, route }: Props) {
     return pools.filter((pool) => {
       const depositTokenInfo = allTokens[pool.dataProps.depositTokenId]
       const isMyPool = new BigNumber(pool.balance).gt(0) && !!depositTokenInfo
-      return activeTab === EarnTabType.MyPools ? isMyPool : !isMyPool
+      return activeTab === EarnTabType.MyPools ? isMyPool : pools
```

Comment:
```suggestion
      return activeTab === EarnTabType.MyPools ? isMyPool : true
```
this is just returning a bool for filtering so should be true always for AllPools case

Reply:
maybe just early bail if tab is all pools? E.g.,
```ts
return activeTab === <all pools> ? pools : pools.filter(...)
```